modules/wkt/src/lib/encode-wkb.ts

```diff
@@ -46,12 +51,16 @@ interface WKBOptions {
  */
 export default function encodeWKB(
   geometry: Geometry | Feature,
-  options: WKBOptions = {hasZ: false, hasM: false}
+  options: WKBOptions | {wkt: WKBOptions}
 ): ArrayBuffer {
   if (geometry.type === 'Feature') {
     geometry = geometry.geometry;
   }
 
+  if ('wkt' in options) {
```

Comment:
I am not sure that we support nested options in the encoders. The Writer/encode... system is not nearly as built out as the Loader/parse... system.

Reply:
This should support both nested and non-nested options as-is